### ISSUE TITLE
Add an AI

### ISSUE THREAD

**Issue by vertti** [tessalt/echo-chamber-js#32]
Adding a chatbot from the Touring competition to this would be awesome. Preferably one on the mean and aggressive side.

**Comment by psiniemi** [comment]
Yeah - an "autotroll" ;) Or something like this: http://www.jabberwacky.com/

**Comment by virtadpt** [comment]
What about a copy of Julia (http://www.personalityforge.com/userviewer.php?UserID=7929)?  After all, trolls and kittehs make the Net go 'round.

**Comment by jevgen** [comment]
I second for AI.

**Comment by kinduff** [comment]
Related #6

**Comment by Hasimir** [comment]
It won't be even vaguely AI-like (but it will be offensive): you could harness [Brython](https://github.com/brython-dev/brython) in order to get my most notorious project (relative to the others, not the world), [FOAD](https://github.com/adversary-org/foad), to respond to users.  It does have a built in random selection mode, but I haven't tested it thoroughly in a while so it  might not play nicely with all of the output forms (some of them screw with certain flags in order to adapt them differently).  Use it or not, it's entirely up to you, the only stipulation not part of any of the licenses is to have fun.  ;)

**Comment by culebron-ala** [comment]
bump

Is there a cheap-ass llm for in-browser? xD
